### PR TITLE
[INLONG-7220][Manager] Optimize OpenStreamSinkController implementation

### LINKED ISSUE #7220: [Improve][Manager] Optimize OpenStreamSinkController implementation
### Description

The parent issue is :https://github.com/apache/inlong/issues/7168

Based on the problems mentioned by the parent class requirements, optimize the implementation of OpenStreamSinkController

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gosonzhang**
- Fixes #7220

Modifications:
1. Most of them add error codes to the exceptions thrown, adjust the logs, including removing the printed logs with clear exceptions, and adjust the log level to debug for the path logs;
2. Adjust the field requirements of the SinkRequest request, and limit the mandatory fields for id-based and key-based updates respectively;
3. Modify 2 Bugs:
  a. For the operation modified by id, the groupId and streamId are not allowed to be modified in the open api:
      image

  b. Correct the judgment on line 93 of HudiSinkOperator.java
      image

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/doris/DorisSinkOperator.java [hunk 1]
```diff
@@ -81,7 +82,10 @@ public StreamSink getFromEntity(@NotNull StreamSinkEntity entity) {
         }
 
         DorisSinkDTO dto = DorisSinkDTO.getFromJson(entity.getExtParams());
-        Preconditions.checkNotEmpty(dto.getFeNodes(), "doris fe nodes is empty");
+        if (dto.getFeNodes() == null) {
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/doris/DorisSinkOperator.java:85]: Suggest to use` StringUtils.isNotBlank`.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java [hunk 2]
```diff
@@ -414,37 +415,61 @@ public Boolean update(SinkRequest request, String operator) {
     @Transactional(rollbackFor = Throwable.class)
     public Boolean update(SinkRequest request, UserInfo opInfo) {
         // check request parameter
-        checkSinkRequestParams(request);
+        // checkSinkRequestParams(request);
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java:418]: Why not delete？

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/doris/DorisSinkOperator.java (hunk 1 above)
Comment (gosonzhang) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/doris/DorisSinkOperator.java]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java (hunk 2 above)
Comment (gosonzhang) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java]: done

**Review by healchow — APPROVED** [review]

**Review by woofyzhao — APPROVED** [review]

**Review by fuweng11 — APPROVED** [review]

**Pull request merged**